src/fmu/ensemble/realization.py

```diff
@@ -1192,6 +1216,14 @@ def get_smryvalues(self, props_wildcard=None):
             a dataframe with values. Raw times from UNSMRY.
             Empty dataframe if no summary file data available
         """
+        warnings.warn(
+            (
+                "realization.get_smryvalues() is deprecated and "
+                "will be removed in fmu-ensemble v2.0.0"
```

Comment:
Add suggestion for alternative? Probably not in much use anyways.
```suggestion
                "realization.get_smryvalues() is deprecated and "
                "will be removed in fmu-ensemble v2.0.0."
                "Use realization.get_smry() instead."
```

Reply:
fixed